pr_agent/algo/utils.py

```diff
@@ -557,7 +557,7 @@ def _fix_key_value(key: str, value: str):
 
 
 def load_yaml(response_text: str, keys_fix_yaml: List[str] = [], first_key="", last_key="") -> dict:
-    response_text = response_text.strip('\n').removeprefix('```yaml').rstrip('`')
+    response_text = response_text.strip().removeprefix("```yaml").removesuffix("```").strip()
```

Comment:
the new logic is not good:

1. you start with a `.strip()` (instead `strip('\n')`), which could be problematic if the completion is an indented code, for example

2. replacing `.rstrip('`')` with `.removesuffix("```")` does make sense

3. `.strip()` in the end - again wrong.

Reply:
you are welcome to open a PR with the removesuffix("```") part

Reply:
Ok fixing now